ParabolInc/parabol
# feat(jira-server): add issue to poker scope

#6158

How to test

- Add Jira Server integration
- Create a poker meeting
- Add a Jira Server issue to the scope
- See it correctly appears in the sidebar
- Click on the issue in the sidebar — see description displayed correctly
